refiner-app/src/config.rs

```diff
@@ -15,7 +17,7 @@ pub struct Config {
 pub struct Refiner {
     pub chain_id: u64,
     pub engine_path: PathBuf,
-    pub engine_account_id: String,
+    pub engine_account_id: ValidatedAccountId,
```

Comment:
By the way, I would prefer to use `#[serde(deserialize_with = '']` here instead of adding a new type.